rusk-recovery/src/state/snapshot/stake.rs

```diff
@@ -0,0 +1,45 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use dusk_bls12_381_sign::PublicKey;
+use rusk_abi::dusk::Dusk;
+use serde_derive::{Deserialize, Serialize};
+
+use super::*;
+
+#[derive(Serialize, Deserialize, PartialEq, Eq)]
+pub struct GenesisStake {
```

Comment:
This shares the same issues of the other API (plus the naming issue I mentioned already).